content/docs/cucumber/api.md

```diff
@@ -848,6 +848,23 @@ The **Command-Line Interface Runner (CLI Runner)** is an executable Java class t
 ```
 java io.cucumber.core.cli.Main
 ```
+Note that you will need to add the following jars to your classpath, in addition to the location of your compiled .class files:
+
+* `cucumber-core-{{% version "cucumberjvm" %}}.jar`
+* `gherkin-{{% version "gherkin" %}}.jar`
+* `cucumber-expressions-{{% version "cucumberjvm" %}}.jar`
+* `datatable-{{% version "datatable" %}}.jar`
+* `datatable-dependencies-{{% version "datatable" %}}.jar`
+* `cucumber-java-{{% version "cucumberjvm" %}}.jar`
+
+Be sure to use the same version for all cucumber-jvm dependencies.  You can find these jars in [Maven Central](https://mvnrepository.com/repos/central).
+You will also need to provide the CLI with your step definitions via the `--glue` option followed by its package name, and the filepath of your feature file(s). 
+
+For example:
+```shell
+java -cp "path/to/each/jar:path/to/compiled/.class/files" io.cucumber.core.cli.Main /path/to/your/feature/files --glue hellocucumber
```

Comment:
If you you use the [Exec Maven plugin](https://www.mojohaus.org/exec-maven-plugin/index.html) you don't have to manage the class path yourself. For example in the skeleton project you can do:

```
mvn exec:java  -Dexec.classpathScope=test -Dexec.mainClass=io.cucumber.core.cli.Main -Dexec.args="--help"
```

This would also avoid needing to list the exact dependencies in the documentation (I don't look forward to keeping that list up to date).

Reply:
I've removed the version numbers and added the bit about the exec plugin -- thanks for the tip, I didn't know that was a workaround.